packages/mermaid/src/diagrams/er/parser/erDiagram.jison

```diff
@@ -26,6 +26,7 @@ accDescr\s*"{"\s*                                { this.begin("acc_descr_multili
 <block>\b((?:PK)|(?:FK)|(?:UK))\b      return 'ATTRIBUTE_KEY'
 <block>(.*?)[~](.*?)*[~]        return 'ATTRIBUTE_WORD';
 <block>[\*A-Za-z_][A-Za-z0-9\-_\[\]\(\)]*  return 'ATTRIBUTE_WORD'
+<block>\`[^`\n]*\`              return 'BACKTICK_STRING';
```

Comment:
Is it possible to add backtick within attribute name as well? I am asking because we are trying to allow basically any characters within attribute name with this PR. Minor

Reply:
Adding one more backtick in the name causes a parse error, and adding a second one breaks it into two attributes. For example: 
```
String `geo.`acc`uracy` 
```
becomes:
![Screenshot 2024-01-22 at 12 10 04 PM](https://github.com/mermaid-js/mermaid/assets/4667097/36d27109-be1c-45f0-92d2-5069084b42d3)

I'm not sure if there is a way around this behavior, other than maybe changing the escape characters from backticks to something else. Let me know if you have suggestions here 🙂

Reply:
They should be escaped then. Common practice is to use escapement character, like `\` (`\\` and `` \` `` respectively for `\` and `` ` ``) or duplicating characters, like `"hello""quotes"` (csv uses that approach), or using another character as wrapper (like double backtick for markdown)

Reply:
I am not sure which approach to pick up, they all have some flaws but it's better to think about such a feature beforehand because it is a real pain to introduce something like this later without braking backwards compatibility